app/config.local.js

```diff
@@ -29,3 +34,4 @@ kubernetes: "dev-mode"
 };
 
 window.MOCK_ORIGIN_SERVICES = false;
+window.MOCK_SERVICES_DATA = true;
```

Comment:
Do we need both flags? Are you looking to get some real data and some mock data in the same app? Is that necessary?